docs/_docs/extensions-and-integrations/change-data-capture-extensions.adoc

```diff
@@ -113,6 +113,14 @@ image:../../assets/images/integrations/CDC-ignite2kafka.svg[]
 | `kafkaRequestTimeout` | Kafka request timeout in milliseconds.  | `3000`
 |===
 
+`kafkaRequestTimeout` property sets how much `IgniteToKafkaCdcStreamer` will wait for `KafkaProducer` to finish request.
+
+NOTE: _Too low value_ of this timeout can lead to frequent failures of `IgniteToKafkaCdcStreamer`
+because of possible delays in network or on Kafka broker.
+Also, proper `KafkaProducer` configuration should be provided, including link:https://kafka.apache.org/documentation/#producerconfigs_request.timeout.ms[request.timeout.ms].
```

Comment:
What is a purpose of this new line? Should this sentence be part of the NOTE or not? If not, please add additional empty line between the NOTE paragraph and the sentence.

Reply:
Good point!